### PR TITLE
Allow use of custom models

### PR DESCRIPTION
**Pull request by faustbrian**
Pretty much just this PR https://github.com/laravel/cashier/pull/337 but for Braintree.

### CODE REVIEW

File: src/Invoice.php [hunk 1]
```diff
@@ -216,7 +216,7 @@ protected function formatAmount($amount)
     public function view(array $data)
     {
         return View::make('cashier::receipt', array_merge(
-            $data, ['invoice' => $this, 'user' => $this->user]
+            $data, ['invoice' => $this, 'owner' => $this->owner]
```
Comment (taylorotwell) [inline src/Invoice.php]: As I said when this was sent to Cashier, you will have to send both variables. This is a breaking change.

**Comment by faustbrian** [comment]
Thanks for merging. Will there be a new release tagged with it soon?

**Comment by arthurkirkosa** [comment]
ETA on a new tag for this? @taylorotwell

**Comment by taylorotwell** [comment]
Done.

**Comment by arthurkirkosa** [comment]
Thanks

**Pull request merged**